Automattic/jetpack
# Site Settings API: Add support for Instant Search flag

Differential Revision: D48122-code

This commit syncs r212314-wpcom.

#### Changes proposed in this Pull Request:

* Adds support for returning the `instant_search_enabled` option as a site setting.

#### Jetpack product discussion

* N/A

#### Does this pull request change what data or activity we track or use?

* N/A

#### Testing instructions:

Use the following Github PR to test this change: https://github.com/Automattic/wp-calypso/pull/44929.

Alternatively, query this API route via WPCOM API console and verify that the `instant_search_enabled` value is being returned in the response.

#### Proposed changelog entry for your changes:

* N/A
